getting-started/README.md

```diff
@@ -0,0 +1,264 @@
+# Getting started developing for Optimism
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)
+
+This tutorial teaches you the basics of Optimism development. 
+Because of [EVM equivalence](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306) the differences are minor, but [they do exist](https://community.optimism.io/docs/developers/build/differences/#).
+
+## Optimism endpoint URL
+
+To access any Ethereum type network you need an endpoint. There are several ways to get one:
```

Comment:
just changing the order here to follow a progression 

There are a few you can use depending on what you will need it for. 

1. For local testing...
2. For limited development testing ... op providers
3. For production providers ...